olive/model/config/io_config.py

```diff
@@ -35,6 +40,10 @@ class IoConfig(ConfigBase):
     # We use a workaround here: first use dim_param like "1" to represent the dimension, and then
     # convert it to int in the onnx model.
     string_to_int_dim_params: List[str] = None
+    # if False, skip kv_cache input
+    # if True, use default KVCacheConfig
+    # if KVCacheConfig, use the provided KVCacheConfig
+    kv_cache_config: Union[bool, KVCacheConfig] = False
```

Comment:
kv_cache better since it might be bool or config? the kv_cache_config seems always be config?